subprojects/testing-base/src/main/java/org/gradle/api/tasks/testing/AbstractTestTask.java

```diff
@@ -167,9 +170,14 @@ protected ListenerManager getListenerManager() {
         return testListenerInternalBroadcaster;
     }
 
-    /**
-     * ATM. for testing only
-     */
+    @Internal
```

Comment:
:x: this annotation is for getters only and should be removed, see its javadoc

Reply:
removed